src/components/view-header/ViewHeader.tsx

```diff
@@ -40,11 +44,16 @@ export type ViewHeaderProps = {
   helpTextKey?: string;
 };
 
+export type ViewHeaderBadge = {
+  id?: string;
+  text?: string | ReactNode;
```

Comment:
No need to provide `string` as a type here, as a string is already a type under `ReactNode`.
```suggestion
  text?: ReactNode;
```